.gitignore

```diff
@@ -1,3 +1,6 @@
+*.log.json
```

Comment:
maybe also `*.log`

Reply:
These custom files and folders may be moved to the bottom.

Reply:
sure